rollbar/__init__.py

```diff
@@ -1648,7 +1648,10 @@ def pop(self):
 
     @property
     def value(self):
-        return self._registry.tags
+        if hasattr(self._registry, 'tags'):
+            return self._registry.tags
+
+        return []
```

Comment:
why or when would be the case where `_LocalTags` is initialized, but does not have a `tags` attribute in `self._registry`?

removing this if condition, fails this test https://github.com/rollbar/pyrollbar/blob/master/rollbar/test/test_rollbar.py#L289

with this:
```
test_report_exception_with_same_exception_as_cause (rollbar.test.test_rollbar.RollbarTest) ... ERROR:rollbar:Exception while reporting exc_info to Rollbar. AttributeError("'thread._local' object has no attribute 'tags'",)
Traceback (most recent call last):
  File "/Users/ajtran/Development/tmp/test-launchdarkly/pyrollbar/rollbar/__init__.py", line 439, in report_exc_info
    return _report_exc_info(exc_info, request, extra_data, payload_data, level=level)
  File "/Users/ajtran/Development/tmp/test-launchdarkly/pyrollbar/rollbar/__init__.py", line 722, in _report_exc_info
    tags = _tags.value + getattr(exc_info[1], '_rollbar_tags', [])[::-1]
  File "/Users/ajtran/Development/tmp/test-launchdarkly/pyrollbar/rollbar/__init__.py", line 1652, in value
    return self._registry.tags
AttributeError: 'thread._local' object has no attribute 'tags'
```

Reply:
I think you should try to get the root exception that is triggering that report and see at which moment that's happening

Reply:
Ah. Ok ok. I think I see what is happening right now. For this test `test_report_exception_with_same_exception_as_cause`, we are running the `report_exc_info` inside a `threading.Thread` and when we do that, `tags` is not set in that `thread._local`, thus this runtime error when trying to call `report_exc_info`. I think I might know a way around this.

Reply:
I've resolved this by initializing `self._registry.tags` in the `append` and `value` method of `_LocalTags`.